app/controllers/api/v2/hostgroups_controller_extensions.rb

```diff
@@ -0,0 +1,70 @@
+module Api
+  module V2
+    # Extends the hostgroups controller to support playing ansible roles
+    module HostgroupsControllerExtensions
+      include ForemanTasks::Triggers
+
+      # TODO: How to add this to API v2 controller?
```

Comment:
Ah I forgot.. if you can move this to the `ForemanAnsible` module that'd be even better. They'd be `ForemanAnsible::Api::V2::HostgroupsController`. Anything similar works. 

The reason is that plugin extensions should have namespaces to avoid clashing with one another:

e.g: foreman_discovery defines HostgroupExtensions, foreman_ansible defines also HostgroupExtensions, now we have two engines with the exact same module name, which could cause problems.